AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -485,13 +499,75 @@ def directional_light(property: str = 'name') -> str:
         """
         Directional Light component properties.
           - 'Camera' an EditorEntity.id reference to the Camera component that controls cascaded shadow view frustum.
-            Must be a different entity than the one which hosts Directional Light component.\n
+            Must be a different entity than the one which hosts Directional Light component.
+          - 'Color' Color of the Light. (
+          - 'Intensity mode' Allows specifying light values in Lux or Ev100.
+          - 'Intensity' Intensity of the light in the set photometric unit.
+          - 'Angular diameter' Angular diameter of the directional light in degrees. The sun is about 0.5.
+          - 'Camera' Entity of the camera for cascaded shadowmap view frustum.
+          - 'Shadow far clip' Shadow specific far clip distance.
+          - 'Shadowmap size' Width/Height of shadowmap.
+          - 'Cascade count' Number of cascades.
+          - 'Automatic splitting' Switch splitting of shadowmap frustum to cascades automatically or not.
+          - 'Split ratio' Ratio to lerp between the two types of frustrum splitting scheme.
+          0 = Uniform scheme which will split the frustum evenly across all cascades.
```

Comment:
nitpick: for the multi-line descriptions, can we slightly indent the additional lines? maybe by 4 spaces or so just so it reads a bit cleaner

Reply:
fixed